pages/common/recsel.md

```diff
@@ -0,0 +1,16 @@
+# recsel
+
+> Print records from a human-editable, plain text database called a recfile.
+> More information <https://www.gnu.org/software/recutils/manual/recutils.html>.
```

Comment:
Perhaps this will read better?
```suggestion
> Print records from a recfile: a human-editable, plain text database.
> See also `guix`.
> More information <https://www.gnu.org/software/recutils/manual/recutils.html>.
```

Reply:
I agree with that :+1: I'll add commands for `guix` later; I noticed there are no tldr pages for that.

Reply:
I've removed the `guix` examples, so now the command uses a "{{data.rec}}" which I think is more clearer. So we won't need *See also `guix`*